cypress/integration/findings_spec.js

```diff
@@ -3,21 +3,213 @@
+    cy.visit(`${Cypress.env('opensearch_dashboards')}/app/${PLUGIN_NAME}#/detectors`);
+
+    cy.wait(10000);
+
+    // Locate Create detector button click to start
+    cy.contains('Create detector').click({ force: true });
+
+    // Check to ensure process started
+    cy.contains('Define detector');
+
+    // Enter a name for the detector in the appropriate input
+    cy.get(`input[placeholder="Enter a name for the detector."]`).type('test detector{enter}');
+
+    // Select our pre-seeded data source (cypress-test-windows)
+    cy.get(`[data-test-subj="define-detector-select-data-source"]`).type(
+      'cypress-test-windows{enter}'
+    );
+
+    // Select threat detector type (Windows logs)
+    cy.get(`input[id="windows"]`).click({ force: true });
+
+    // Wait for detector rules to load - timeout on click above ineffective
+    cy.wait(10000);
+
+    // Click Next button to continue
+    cy.get('button').contains('Next').click({ force: true }, { timeout: 2000 });
+
+    // Check that correct page now showing
+    cy.contains('Required field mappings');
+
+    // Select appropriate names to map fields to
+    for (let field_name in TEST_FIELD_MAPPINGS) {
+      const mappedTo = TEST_FIELD_MAPPINGS[field_name];
+
+      cy.contains('tr', field_name).within(() => {
+        cy.get(`[data-test-subj="detector-field-mappins-select"]`).select(mappedTo);
+      });
+    }
+
+    // Continue to next page
+    cy.get('button').contains('Next').click({ force: true }, { timeout: 2000 });
+
+    // Check that correct page now showing
+    cy.contains('Set up alerts');
+
+    // Type name of new trigger
+    cy.get(`input[placeholder="Enter a name for the alert condition."]`).type('test_trigger');
+
+    // Type rule name to trigger alert
+    cy.get(`[data-test-subj="alert-rulename-combo-box"]`).type('USB Device Plugged{enter}');
+
+    // Select applicable severity levels
+    cy.get(`[data-test-subj="security-levels-combo-box"]`).click({ force: true });
+    cy.contains('1 (Highest)').click({ force: true });
+
+    // Type rule severity to trigger alert
+    cy.get(`[data-test-subj="alert-severity-combo-box"]`).type('low{enter}');
+
+    // Continue to next page
+    cy.contains('Next').click({ force: true });
+
+    // Confirm page is reached
+    cy.contains('Review and create');
+
+    // Confirm field mappings registered
+    cy.contains('Field mapping');
+    for (let field in TEST_FIELD_MAPPINGS) {
+      const mappedTo = TEST_FIELD_MAPPINGS[field];
+
+      cy.contains(field);
+      cy.contains(mappedTo);
+    }
+
+    // Create the detector
+    cy.get('button').contains('Create').click({ force: true });
+
+    cy.wait(10000);
   });
 
-  //TODO - get findings to be present, then test
+  it('displays findings based on recently ingested data', () => {
+    // Visit Findings page
+    cy.visit(`${Cypress.env('opensearch_dashboards')}/app/${PLUGIN_NAME}#/findings`);
+    cy.wait(10000);
```

Comment:
You're essentially just waiting for the table to finish loading, correct? If so, this `wait` could be replaced with `cy.contains('No items found', { timeout: 20000 }).should('not.exist')`. That will make the test wait for up to 20 seconds for the empty table text to be replaced with the table rows.

Reply:
for the comment on line27,  decreased to 5000 to let page load.  Force click works here, will try to reduce/eliminate waits elswhere.

Reply:
done as much as cypress will allow

Reply:
done